datafusion/core/src/physical_plan/file_format/parquet.rs

```diff
@@ -98,6 +100,7 @@ impl ParquetExec {
             MetricBuilder::new(&metrics).global_counter("num_predicate_creation_errors");
 
         let pruning_predicate = predicate
+            .clone()
```

Comment:
the key difference here is that the original predicate is kept, even if the creation of the pruning predicate fails